enhancements/oc/mirroring-manifests.md

```diff
@@ -105,78 +98,104 @@ mirror
 
-FIXME
+Currently a cluster upgrade can be accomplished on a cluster that does not have an active connection to the internet.
+However manual steps are required to create a ConfigMap containing the signature data required for update image verification.
+This enhancement will automatically create the ConfigMap from the image manifest files.
 
 #### In-cluster Cincinnati
 
 FIXME
 
 ### Risks and Mitigations
 
-FIXME
+One risk considered is that a user familiar with the current behavior of the `oc adm release mirror ...` command may inadvertently apply manifests to thier cluster.
+This has been addressed by requiring the explicit `--apply-manifests` argument to apply manifests to a cluster.
 
 ## Design Details
 
 ### Test Plan
 
-FIXME
+The current e2e test, that makes sure the command always exits successfully and that certain apsects of the content are always present, will be modified to validate the new functioanlity as well.
 
 ### Graduation Criteria
 
-FIXME
-
 ### Upgrade / Downgrade Strategy
 
-FIXME
+The image is included in the payload, but has no content running in a cluster to upgrade.
 
 ### Version Skew Strategy
 
-The proposal for [applying directly to the target cluster](#applying-directly-to-the-target-cluster) adds a new dependency on the cluster to which `oc` connects for Kubernetes activity.
-The explicit `--apply-manifests` argument requires users who have been using the previous `oc adm release mirror` implementation (which did not push manifests) to explicitly opt in to the new functionality, so there is no change that they accidentally pushing manifests into the wrong cluster.
-FIXME: check to see how this works if there are conflicts but neither `--force-conflicts` not `--overwrite` was set.
+The proposal for [applying manifests directly to the target cluster](#applying-manifests-directly-to-the-target-cluster) adds a new dependency on the cluster to which `oc` connects for Kubernetes activity.
+The explicit `--apply-manifests` argument requires users who have been using the previous `oc adm release mirror` implementation (which did not push manifests) to explicitly opt in to the new functionality, so there is no chance that they accidentally push manifests into the wrong cluster.
 
 ## Implementation History
 
-FIXME
-
 ## Drawbacks
 
-FIXME
-
 ## Alternatives
 
-FIXME
-
 ## Infrastructure Needed [optional]
 
-Use this section if you need things from the project. Examples include a new
-subproject, repos requested, github details, and/or testing infrastructure.
-
-Listing these here allows the community to get the process for these resources
-started right away.
-
 [cvo-config-map-signatures]: https://github.com/openshift/cluster-version-operator/pull/279
-[custom-resource]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
+[custom-resources]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extens`oc adm release mirror ...` commandion/custom-resources/
```

Comment:
Is `oc adm release mirror ...` an accidental copy/paste issue?

Reply:
yes